tests/topology/test_pbc_cell_div.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Sep 12 18:49:53 2019
+
+@author: georgecourcoubetis
+"""
+
+import time
+
+start = time.time()
+
+from tyssue import stores
+from pathlib import Path
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
We don't need to plot anything in the test, just to check that the test functions run